### PR TITLE
Telephone is More...Ring-y

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Telephone can now be activated in hand to make a ring noise.

Force on telephones reduced from 3 to 0 (now you can call people any time without threat of serious injury!)

:cl: Fox McCloud
add: Telephones will now ring when activated in-hand
tweak: Force on telephones removed
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/misc.dm [hunk 1]
```diff
@@ -127,10 +127,19 @@
...
+	force = 0
+	throwforce = 2
 	throw_speed = 1
 	throw_range = 4
 	w_class = 2
 	attack_verb = list("called", "rang")
-	hitsound = 'sound/weapons/ring.ogg'
\ No newline at end of file
+	hitsound = 'sound/weapons/ring.ogg'
+	var/on_cooldown = 0
```
Comment (PJB3005) [inline code/game/objects/items/weapons/misc.dm]: World time timestamps are safer and more sane.

**Comment by PJB3005** [comment]
\>ruining a portal reference.

**Comment by Fox-McCloud** [comment]
the concept of the red phone far predates portal: https://en.wikipedia.org/wiki/Moscow%E2%80%93Washington_hotline

**Comment by taukausanake** [comment]
What does this phone actually do? I never messed with it

**Comment by Fox-McCloud** [comment]
It rings when you attack someone with it---that's literally it.

**Comment by Spacemanspark** [comment]
I personally don't like removing the damage on it, it's always funny to beat someone to death with a phone while shouting "IT'S FOR YOU!".
Other than that, :+1:

**Comment by Iamgoofball** [comment]
@Fox-McCloud we actually had an argument about it awhile ago at tg, it was added as a portal reference and the guy forgot about the washington-moscow connector

**Comment by Spacemanspark** [comment]
Wasn't that your PR to call CC using the phone, goofball?

**Comment by Iamgoofball** [comment]
yeah

**Comment by Midaychi** [comment]
Can you give it a decent throwforce and make it ring when you throw it? That way you can use it in pneumatic cannons as a weapon. Give it a tiny embed chance too.

File: code/game/objects/items/weapons/misc.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/objects/items/weapons/misc.dm]: ya this

**Comment by RemieRichards** [comment]
@Fox-McCloud This Red Phone was added by Giacom, and it *IS* a Portal Reference, He confirmed it when we had the big argument goof mentioned I was the only person to know it was a Portal Ref.

don't ruin it.

**Comment by Zilenan91** [comment]
The actual red phone reference in Portal is that it's a literal red phone someone sat next to and was supposed to call something on it if Glados ever became sentient and dangerous. This is also why there's a red phone in the AI core on Metastation.

**Comment by TullyBurnalot** [comment]
The potential for jokes here is immense. :100:

**Comment by fludd12** [comment]
remove cooldown

we must ring every second

:phone:

**Comment by tigercat2000** [comment]
would rather the force is left intact, it's a phone and it's base station

**Pull request merged**